apps/curbside_concerts/lib/curbside_concerts/requests.ex

```diff
@@ -116,6 +116,13 @@ defmodule CurbsideConcerts.Requests do
     |> Repo.one()
   end
 
+  def find_request_with_children(request_id) do
+    Request
+    |> where([r], r.id == ^request_id)
+    |> preload([{:session, :musician}, :genres])
+    |> Repo.one()
+  end
+
```

Comment:
The original intention of this method was to retrieve the fully loaded Request that contains the session, musician and genre.  I wanted to keep it separate from the `find_request` method in case to not modify it since other parts of the app are using it. 

However, this was all before I merged master into this branch and see that you've already made that change 😄 so I'll remove this method 👍